src/view/placeholder/placeholder.jsx

```diff
@@ -9,24 +10,26 @@ type Props = {|
 export default class Placeholder extends PureComponent<Props> {
   render() {
     const placeholder: PlaceholderType = this.props.placeholder;
-    const { borderBox, margin, display, tagName } = placeholder;
+    const client: BoxModel = placeholder.client;
+    const display: string = placeholder.display;
+    const tagName: string = placeholder.tagName;
```

Comment:
destructure these?

Reply:
I wanted to give the types explicitly - especially for the `client: BoxModel` so that it is clear